static/skin/taskbar.js

```diff
@@ -5,24 +5,38 @@ function htmlDecode(input) {
 
 (function ($) {
     const root = $( `link[type='root']` ).attr("href");
+    const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
 
     $( "#kiwixsearchbox" ).autocomplete({
 
-        source: `${root}/suggest?content=${window.location.pathname.split(`${root}/`)[1].split('/')[0]}`,
+        source: `${root}/suggest?content=${bookName}`,
         dataType: "json",
         cache: false,
 
         response: function( event, ui ) {
 
               for(const item of ui.content) {
                   item.label = htmlDecode(item.label);
-                  item.value = htmlDecode(item.value);
+                  item.value = encodeURI(htmlDecode(item.value));
+
+                  if (item.label === `containing '${item.value}'...`) {
```

Comment:
@veloman-yunkan would it be better if I don't place this check for all `item.label` and just check the label of last one if it is of this form then change it's URL?